linux-anvil/Dockerfile

```diff
@@ -46,34 +47,41 @@ RUN curl -s -L -O https://github.com/krallin/tini/releases/download/v0.10.0/tini
     chmod +x tini && \
     mv tini /usr/local/bin
 
+# Add a file for users to source to activate the `conda`
+# environment `root` and the devtoolset compiler. Also
+# add a file that wraps that for use with the `ENTRYPOINT`.
+COPY entrypoint_source /opt/docker/bin/entrypoint_source
+COPY entrypoint /opt/docker/bin/entrypoint
```

Comment:
This is the main reason I was suggesting leaving `conda` where it is. Basically this becomes a little odd sourcing a script in `/opt` and having `conda` in `$HOME`.